.circleci/config.yml

```diff
@@ -69,6 +74,23 @@ jobs:
             docker load -i ./docker-images/spring-cloud-kubernetes-configuration-watcher.tar
             docker load -i ./docker-images/spring-cloud-kubernetes-discoveryserver.tar
             docker load -i ./docker-images/spring-cloud-kubernetes-configserver.tar
+      - run:
```

Comment:
so it seems that there is a way to drop run.sh file that we currently have... I'll see what happens with the rest of modules.